### PR TITLE
Use new method for obtaining DB dialect

### PR DESCRIPTION
**Pull request by mabartos**
While investigating another issue, I noticed a comment and created this minor fix :)

@vramik Could you please check it? Thanks

### CODE REVIEW

**Review by vramik — DISMISSED** [review]
LGTM, thanks @mabartos for spotting this and providing the PR. 

One **optional** suggestion from me would be to remove the `private Dialect getDialect(FunctionContributions)` completely and use `fc.getDialect()` directly on line 61.

File: model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/hibernate/contributor/JpaMapFunctionContributor.java [hunk 1]
```diff
@@ -53,9 +53,8 @@ private BasicType getBasicType(FunctionContributions fc, BasicTypeReference<?> b
         return fc.getTypeConfiguration().getBasicTypeRegistry().resolve(btr);
     }
 
-    // Since Hibernate 6.2.0 dialect should be accessible via FunctionContributions.getDialect(): https://github.com/hibernate/hibernate-orm/pull/5964/
     private Dialect getDialect(FunctionContributions fc) {
```
Comment (jonkoops) [inline model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/hibernate/contributor/JpaMapFunctionContributor.java:56]: At this point, why even have this method? Might as well call it directly on `fc` wherever it's used.

**Review by jonkoops — APPROVED** [review]
LGTM

**Review by vramik — APPROVED** [review]
Thanks @mabartos

**Review by hmlnarik — APPROVED** [review]

**Pull request merged**